R/MeasureFairness.R

```diff
@@ -0,0 +1,54 @@
+#' @title Fairness Measure
+#' @name mlr_measures_fairness
+#'
+#' @description
+#' This measure specializes [Measure] for Fairness Measure problems:
+#' Users could use [groupwise_abs_diff()], [groupwise_diff()], or [groupwise_quotient()] to evaluate
+#' the fairness measures based on base_measures. For example, the false positive rate bias or
+#' the equalized odds ratios. More examples could viewed on mlr3fairness wikipage.
+#'
+#' Predefined measures can be found in the [dictionary][mlr3misc::Dictionary] [mlr_measures].
+#' Predefined operations can be found in the [mlr3fairness].
+#'
+#' @export
+MeasureFairness = R6Class("MeasureFairness", inherit = Measure, cloneable = FALSE,
+  public = list(
+    #' @template field_fun
+    fun = NULL,
+
+    #' @template field_base_measure
+    base_measure = NULL,
+
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    #' @param base_measure The measure used to perform fairness operations.
```

Comment:
Stating the argument type ([Measure] in this case) would help. We try to always do this, c.f. mlr3 man pages.

Reply:
Corrected

Reply:
> Corrected

Did you forget to push? Where did you correct it?

Reply:
I haven't push it yet, should be up here now.